packages/utilities/psammead-storybook-helpers/src/input-provider.jsx

```diff
@@ -4,14 +4,23 @@ import { Helmet } from 'react-helmet';
 import * as scripts from '@bbc/gel-foundations/scripts';
 import LANGUAGE_VARIANTS from './text-variants';
 
-const inputProvider = (slots, componentFunction, services) => () => {
+const inputProvider = (
+  slots,
+  componentFunction,
+  services,
+  options = {},
```

Comment:
are we expecting more options? what if this arg was just `defaultService = 'news'`

Reply:
Yeah, ide almost prefer 
```
(slots, componentFunction, services)
``` -> 
```
({ slots, componentFunction, services, defaultService })
```
or put `services` inside options, as its optional

Reply:
Feel 3 args is already too many - you can see the problem in the tests where you have to pass `null` as 2 arguments

Reply:
@dr3 agree that all the arguments should just be one object - that's a big refactor though that will need an update to all consumers - not sure that's in scope for media pod?  We just want our page on storybook :'(

Reply:
https://github.com/ryanmcdermott/clean-code-javascript#function-arguments-2-or-fewer-ideally

Reply:
@ryanmccombe definitely not in scope for the media pod or this PR really. Could you raise an issue about that refactor though so it can be picked up and looked into by the core pod at a later date.

Reply:
https://github.com/bbc/simorgh/issues/3077